src/utils/sourcemaps.rs

```diff
@@ -814,6 +839,11 @@ impl SourceMapProcessor {
                     }
                 };
 
+                inject::insert_empty_mapping(&mut sourcemap_file.contents).context(format!(
```

Comment:
Create `Sourcemap` struct and pass it through `insert_empty_line`, then `fixup` and then finally serialize and store in the original file.

Reply:
`fixup_sourcemap` needs access to the bytes to hash them into a debug id. I think I'll just leave it as it is.